styleguidekit-assets-altinn/dist/styleguide/css/styleguide.css

```diff
@@ -223,7 +223,7 @@
   */
 }
 .sg-acc-handle:after, .a-sg-switch-dropdown > span:after {
-  content: '▼';
+  content: 'Ôû╝';
```

Comment:
Dette ser litt rart ut. Burde vi finne en utf eller ascii kode for denne pilen nedover, eller i hvertfall forsikre oss om at den blir riktig når det bygges?